lib/airrecord/table.rb

```diff
@@ -197,6 +197,10 @@ def serializable_fields(fields = self.fields)
       }]
     end
 
+    def ==(other)
+      serializable_fields == other.serializable_fields
```

Comment:
I updated this to also reflect that the two classes are equal too, misstake on my side.